src/runtime/auto-prompt-manager-test.js

```diff
@@ -4348,30 +4344,6 @@ describes.realWin('AutoPromptManager', (env) => {
         .resolves(clientConfig)
         .once();
       getArticleExpectation = entitlementsManagerMock.expects('getArticle');
-      getArticleExpectation
```

Comment:
perhaps we should add a rewarded ad case for the non-local flow test?

Reply:
Added a test to verify that it calls into the non-local flow.